src/components/NotesMindMapView/NotesMindMapViewComponent.js

```diff
@@ -88,24 +92,45 @@ export class NotesMindMapViewComponent extends Component {
     let targetNote = noteStorage.findNoteById(targetNoteId);
 
     if (!targetNote) {
+      console.log("available");
+      console.log(this.props.changeVisNetworkNote);
       throw new Error('changeVisNetworkNote(): couldn\'t find targetNote in noteStorage by id: ', targetNoteId);
     }
 
     this.props.changeVisNetworkNote({ targetNote, visNetwork });
   };
 
   visNetworkClickHandler = event => {
-    this.closeNoteNameEditor();
-    if (VisNetworkHelper.clickedOnNote(event)) {
+    const { currentViewedNote } = this.state;
+    const { changeNoteParentMode } = this.props;
+
+    if(!changeNoteParentMode) 
```

Comment:
Please conform to the current code style: there should be a space between `if` and parentheses.